fetch-versions.sh

```diff
@@ -70,11 +70,11 @@ for project in "${projects[@]}"; do
     file=${activiti_core_file}
     ;;
   'activiti-cloud')
-    file=repos-activiti-cloud-mono.txt
-    ;;
-  'activiti-cloud-dependencies')
     file=repos-activiti-cloud.txt
-    examples_file=repos-activiti-cloud-examples.txt
+    ;;
+  'activiti-cloud-application')
+    file=repos-activiti-cloud-application.txt
+#    examples_file=repos-activiti-cloud-examples.txt
```

Comment:
any reason why we leave it commented out?

Reply:
good question, I'm reviewing

Reply:
done

Reply:
good 👍